### DISCUSSION TITLE
请教各位大神：Alist如何添加B2云存储

### DISCUSSION THREAD

**Discussion by wotime** [AlistGo/alist#1648] [Q&A]
我想将B2云存储的存储桶添加到Alist，但不知如何操作，一直失败，向各位大神请教如何添加

**Comment by IMZCC** [comment]
新建对象存储即可
![image](https://user-images.githubusercontent.com/25655983/201575176-fd06f3ef-9f34-4999-a4c5-e02bc0b641d8.png)
